lib/brute/http_basic_auth/engine.py

```diff
@@ -92,18 +105,16 @@ def login(user, passwd, target, port, timeout_sec, log_in_file, language, retrie
         return flag
 
 
-def check_auth(target, timeout_sec, language, port):
+def check_auth(target, timeout_sec, language, port, headers):
```

Comment:
please do not pass headers as parameters as this is breaking the core framework - headers are not parameters which are passed from the core